doc/features/currencies.rst

```diff
@@ -49,6 +49,36 @@ The Bitcoin currencies provides a single currency: the Bitcoin. It uses XBT as i
     $currencies->contains(new Currency('USD')); // returns boolean whether USD is available in this repository (false)
     $currencies->subunitFor(new Currency('XBT')); // returns the subunit for the Bitcoin (8)
 
+CurrencyList
+-----------------
+
+The CurrencyList class provides a way for the developer to instantiate his own custom currencies. When initializing
```

Comment:
I would rephrase this paragraph like this:

The CurrencyList class provides a way for a developer to build a custom currency repository.
The class accepts an associative array indexed by the currency code
and another associative array as the value containing `minorUnit` and `numericCode` keys and positive integer values.